python/cudf/cudf/tests/test_replace.py

```diff
@@ -351,3 +351,14 @@ def test_series_with_nulls_where(fill_value):
     expect = psr.where(psr == 0, fill_value)
     got = sr.where(sr == 0, fill_value)
     assert_eq(expect, got)
+
+
+def test_series_multiple_times_with_nulls():
+    sr = Series([1, 2, 3, None])
+    expected = Series([None, None, None, None], dtype=np.int64)
+
+    for i in range(10):
+        got = sr.replace([1, 2, 3], None)
+        # This print is required as it will fail only if there is print
```

Comment:
Why run the exact same test 10 times? And it doesn't make sense that the print is required.

Reply:
When print is used along with the loop then only it is reproducible as of now, and it starts to occur from 3rd iteration, so I have modified the iteration to 3.

Reply:
Let's not rely on behavior we don't understand. While I don't fully understand what you are saying, it sounds like there's a but here.

Reply:
I agree with @harrism here. Is there any way we can repro this without a print statement? This looks like a janky test.